yt_dlp/extractor/netverse.py

```diff
@@ -149,28 +150,54 @@ def _real_extract(self, url):
 
 class NetversePlaylistIE(NetverseBaseIE):
     _VALID_URL = r'https?://(?:\w+\.)?netverse\.id/(?P<type>webseries)/(?P<display_id>[^/?#&]+)'
-    _TEST = {
+    _TESTS = [{
+        # multiple season
         'url': 'https://netverse.id/webseries/tetangga-masa-gitu',
         'info_dict': {
             'id': 'tetangga-masa-gitu',
             'title': 'Tetangga Masa Gitu',
         },
-        'playlist_count': 46,
-    }
+        'playlist_count': 519,
+    }, {
+        # single season
+        'url': 'https://netverse.id/webseries/kelas-internasional',
+        'info_dict': {
+            'id': 'kelas-internasional',
+            'title': 'Kelas Internasional',
+        },
+        'playlist_count': 203,
+    }]
 
-    def parse_playlist(self, url, page_num):
-        _, playlist_json = self._call_api(url, query={'page': page_num + 1})
-        for slug in traverse_obj(playlist_json, ('response', 'related', 'data', ..., 'slug')):
+    def parse_single_season_playlist(self, page_num, url=None, slug=None, custom_id=None, season_id='',
+                                     endpoint=None):
+
+        _, playlist_json = self._call_api(
+            url=url, slug=slug, query={'page': page_num + 1}, season=season_id, custom_id=custom_id,
+            endpoint=endpoint)
+        for slug in traverse_obj(playlist_json, ('response', ..., 'data', ..., 'slug')):
             yield self.url_result(f'https://www.netverse.id/video/{slug}', NetverseIE)
```

Comment:
This really isn't complicated enough to warrant it's own function imo. Especially considering how all the vars has to be passed in from parse_playlist